packages/inngest/src/next.ts

```diff
@@ -36,6 +36,22 @@ import { type SupportedFrameworkName } from "./types";
  */
 export const frameworkName: SupportedFrameworkName = "nextjs";
 
+/**
+ * The shape of a request handler, supporting Next.js 12+.
+ *
+ * We are intentionally abstract with the arguments here, as Next.js's type
+ * checking when building varies wildly between major versions; specifying
+ * different types (even optional types) here can cause issues with the build.
+ *
+ * This change was initially made for Next.js 15, which specifies the second
+ * argument as `RouteContext`, whereas Next.js 13 and 14 omit it and Next.js 12
+ * provides a `NextApiResponse`, which is varies based on the execution
+ * environment used (edge vs serverless).
+ */
+export type RequestHandler = (
+  ...args: [expectedReq: NextRequest, res: unknown]
```

Comment:
Maybe I'm dumb, but what does `...args: [expectedReq: NextRequest, res: unknown]` do that `NextRequest, res: unknown` doesn't?

Reply:
True! Should be able to ditch that after adding `unknown`.